QUICKSTART.md

```diff
@@ -0,0 +1,36 @@
+# LAMP stack development environment quickstart
```

Comment:
Was just assuming based on the PR title and the D8 specific comments here.  But yes, generic Drupal is fine by me.